packages/core/state/selection/logics.ts

```diff
@@ -709,23 +704,29 @@ const replaceDataSourceLogic = createLogic({
 // If removing, hides pop-up (if there is one) and unsets bool
 const setDataSourceReloadErrorLogic = createLogic({
     async process(deps: ReduxLogicDeps, dispatch, done) {
-        const { payload: dataSourceName, type } = deps.action;
-        const datasourceErrorDefaultMessage = `There was an error loading the data source file
-                        '${dataSourceName}'. Please re-select the data source file or a replacement.
-                    </br>
-                        If this is a local file, the browser&apos;s permissions to access the file
-                        may have expired since last time. If so, consider putting the file in a
-                        cloud storage and providing the URL to avoid this issue in the future.`;
-
-        if (type === ADD_DATASOURCE_RELOAD_ERROR) {
+        const isNewError = deps.action.type === ADD_DATASOURCE_RELOAD_ERROR;
```

Comment:
I'm running into an issue on staging where no matter what's wrong with the file it only shows the "Lost access to the data source" error. This screenshot is from when I uploaded a csv with no file path
<img width="762" alt="image" src="https://github.com/user-attachments/assets/bc06621c-f7ae-4e4e-9551-a5a69a7b0179">

Reply:
This is what you should be seeing and is what I am seeing :( I'll poke around
<img width="851" alt="Screen Shot 2024-11-11 at 9 30 40 AM" src="https://github.com/user-attachments/assets/bbd4df14-3cb9-40a3-be64-c5e23e6f0a65">

Reply:
I'm trying again this morning and it's working about half the time. I can't quite nail down what the condition are where it shows the correct error vs the one I have screen captured

Reply:
Fixed in https://github.com/AllenInstitute/biofile-finder/pull/321/commits/91fc03ad9d70ddf7dbad18723abca82c5b117de1